package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "foreman_ansible",
+  "version": "1.0.0",
+  "description": "[![Code Climate](https://codeclimate.com/github/theforeman/foreman_ansible/badges/gpa.svg)](https://codeclimate.com/github/theforeman/foreman_ansible) [![Gem Version](https://badge.fury.io/rb/foreman_ansible.svg)](https://badge.fury.io/rb/foreman_ansible) [![GPL License](https://img.shields.io/github/license/theforeman/foreman_ansible.svg)](https://github.com/theforeman/foreman_ansible/blob/master/LICENSE)",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
```

Comment:
If you could please add eslint & hound here, it'd be great to ensure the code standards are followed from day 1.

https://github.com/theforeman/foreman_remote_execution/pull/130 for an example of that